sphinx/index.rst

```diff
@@ -69,5 +69,7 @@ Other
 .. toctree::
    :hidden:
    :name: hidden-index
+   :glob:
 
    terms
+   course/*
```

Comment:
This creates the `course` directory in which I've put the UM-UT course outline, it won't be visible from the index page but we can link users to it. Will make a note to pass the link to this page onto the UM-UT course coordinators once it is live.